src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+            addFormNumericBox(F("Ref EC"), F("Plugin_103_ref_cal_L"), PCONFIG(4));
+            addUnit(F("&micro;S"));
+
+            addRowLabel(F("<strong>High calibration</strong>"));
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_H"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_H\").onclick=function() {document.getElementById(\"Plugin_103_enable_cal_dry\").checked = false;document.getElementById(\"Plugin_103_enable_cal_single\").checked = false;;document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;};</script>"));
+            addFormNumericBox(F("Ref EC"), F("Plugin_103_ref_cal_H"), PCONFIG(5));
+            addUnit(F("&micro;S"));
+
+            status = _P103_send_I2C_command(I2Cchoice, F("Cal,?"), sensordata);
+
+            if (status) {
+                switch (sensordata[5]) {
+                case '0':
+                    addFormNote(F("<span style='color:red'>Calibration needed</span>"));
+                    break;
+                case '1':
+                    addFormNote(F("<span style='color:green'>Single point calibration ok</span>"));
+                    break;
+                case '2':
+                    addFormNote(F("<span style='color:green'>Two points calibration ok</span>"));
+                    break;
+                }
+            }
+        }
+        break;
+      }
+
+        // Temperature compensation
+      if(board_type == PH || board_type == EC) {
+        
+        addFormSubHeader(F("Temperature compensation"));
+        char deviceTemperatureTemplate[40];
+        LoadCustomTaskSettings(event->TaskIndex, (byte *)&deviceTemperatureTemplate, sizeof(deviceTemperatureTemplate));
+            addFormTextBox(F("Temperature "), F("Plugin_103_temperature_template"), deviceTemperatureTemplate, sizeof(deviceTemperatureTemplate));
+            addFormNote(F(
+                            "You can use a formulae and idealy refer to a temp sensor (directly, via ESPEasyP2P or MQTT import) ,e.g. '[Pool#Temperature]'. If you don't have a sensor, you could type a fixed value like '25' for '25.5'."));
+            double value;
+            char strValue[5];
+            String deviceTemperatureTemplateString(deviceTemperatureTemplate);
+            String pooltempString(parseTemplate(deviceTemperatureTemplateString, 40));
+            addHtml(F("<div class='note'>"));
+
+            if (Calculate(pooltempString.c_str(), value) != CalculateReturnCode::OK) {
+                addHtml(F("It seems I can't parse your formulae. Fixed value will be used!"));
```

Comment:
Is the note class div also needed here if the `Calculate` function returns OK?

Reply:
I will check this